source/MenuPanel.cpp

```diff
@@ -157,7 +158,11 @@ bool MenuPanel::KeyDown(SDL_Keycode key, Uint16 mod, const Command &command, boo
 		// If no player is loaded, the "Enter Ship" button becomes "New Pilot."
 		// Request that the player chooses a start scenario.
 		// StartConditionsPanel also handles the case where there's no scenarios.
-		GetUI()->Push(new StartConditionsPanel(player, gamePanels, GameData::StartOptions(), nullptr));
+		std::vector<StartConditions> visibleStarts;
+		for(const auto &start : GameData::StartOptions())
+			if(start.Visible(GameData::GlobalConditions()))
+				visibleStarts.emplace_back(start);
```

Comment:
Redundant with what the StartConditionsPanel constructor already does. Should be removed, with the start panel having the responsibility of determining which starts to show.